api/api/src/test/scala/uk/ac/wellcome/platform/api/services/ElasticsearchServiceTest.scala

```diff
@@ -229,6 +240,147 @@ class ElasticsearchServiceTest
     }
   }
 
+  describe("searches using JustBoostQuery") {
+    it("finds results for a JustBoostQuery search") {
+      withLocalWorksIndex { index =>
+        // Longer text used to ensure signal in TF/IDF
+        val text = "Text that contains Aegean"
+
+        val work = createIdentifiedWorkWith()
+        val matchingTitle =
+          createIdentifiedWorkFrom(work)(title = f"$text title")
```

Comment:
Another way to write this that might be a little clearer (and save you defining a method with a stack of arguments) would be:

```scala
val matchingTitle = work.copy(title = s"$text title")
```

Also I’m reasonably sure f-strings are Python, not Scala, no?

Reply:
🐍

Reply:
I don't think copy is the same because `createIdentifiedWorkFrom` is a shorthand for
`val matchingTitle = work.copy(canonicalId: String = createCanonicalId, title = s"$text title")`
not
`val matchingTitle = work.copy(title = s"$text title")`

However, it is okay with random works if the query text is long enough so I have changed it.

`f` [formatted strings](https://docs.scala-lang.org/overviews/core/string-interpolation.html) are valid Scala though the expressiveness isn't needed here, so have removed that too.